util/memory/tracker.go

```diff
@@ -712,6 +712,28 @@ func (t *Tracker) setParent(parent *Tracker) {
 	t.parMu.parent = parent
 }
 
+// CountAllChildrenMemUse return memory used tree for the tracker
+func (t *Tracker) CountAllChildrenMemUse() map[string]int64 {
+	trackerMemUseMap := make(map[string]int64, 1024)
+	countChildMem(t, "", trackerMemUseMap)
+	return trackerMemUseMap
+}
+
+func countChildMem(t *Tracker, familyTreeName string, trackerMemUseMap map[string]int64) {
+	familyTreeName += "[" + strconv.Itoa(t.Label()) + "] <- "
```

Comment:
```
if len(familyTreeName) > 0 {
        familyTreeName += " <- "
}
familyTreeName += "[" + strconv.Itoa(t.Label()) + "]"
```
And remove explain.goL201-205, the code is better understand

Reply:
> ```
> if len(familyTreeName) > 0 {
>         familyTreeName += " <- "
> }
> familyTreeName += "[" + strconv.Itoa(t.Label()) + "]"
> ```
> 
> And remove explain.goL201-205, the code is better understand

done.